core/src/main/java/tachyon/worker/hierarchy/StorageDir.java

```diff
@@ -117,6 +128,7 @@ public void accessBlock(long blockId) {
     synchronized (mLastBlockAccessTimeMs) {
       if (containsBlock(blockId)) {
         mLastBlockAccessTimeMs.put(blockId, System.currentTimeMillis());
+        updateReferenceFrequency(blockId, REFERENCE_FREQUENCY_INCREMENT);
```

Comment:
Instead of using a special flag -1 to mean increment, will adding anohter method IncrementReferenceFrequency(blockId) be easier to understand ? Then the method updateReferenceFrequency can also be simplified. When a block is added the first time, the reference number can be updated to an initial reference number which is 1.